wallettool/src/main/java/org/bitcoinj/wallettool/WalletTool.java

```diff
@@ -186,7 +182,7 @@ public class WalletTool implements Callable<Integer> {
     private String pubKeyStr;
     @CommandLine.Option(names = "--privkey", description = "Specifies a WIF-, hex- or base58-encoded private key.")
     private String privKeyStr;
-    @CommandLine.Option(names = "--addr", description ="Specifies a Bitcoin address, either segwit or legacy.")
+    @CommandLine.Option(names = "--addr", description = "Specifies a Bitcoin address, either segwit or legacy.")
```

Comment:
Can you please make a separate commit with these whitespace error corrections (these two and any others you can find)?